src/s1/s1/static/stylesheets/scss/components/_base_theme.scss

```diff
@@ -1,4 +1,3 @@
 html {
-    @include rem-baseline;
```

Comment:
This property was a duplicate (also in core) so I removed it. I agree that it does not really belong into this commit.